spec/models/project_spec.rb

```diff
@@ -0,0 +1,29 @@
+require 'rails_helper'
+
+describe Project do
+  describe ".calculate_leaderboard" do
+    let!(:project) { create :project }
+
+    context 'without entries' do
+      it 'is not generating the leaderboard' do
+        expect {
```

Comment:
@schmierkov please use do/end for multiline blocks. https://github.com/bbatsov/ruby-style-guide#single-line-blocks